samples/UrlHelperSample/wwwroot/web.config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
But since its a sample we tend to use it in VS and if we do not check this in we would always see a pending change to be committed.